wallet/lib/pages/detailed_asset_view/owner_view.dart

```diff
@@ -508,11 +512,23 @@ class _OwnerBottomDrawerState extends State<OwnerBottomDrawer> {
                                         ),
                                       ),
                                       SizedBox(
-                                        height: 20.h,
-                                      ),
-                                      SizedBox(
-                                        height: 20.h,
+                                        height: 12.h,
                                       ),
+                                      if (viewModel.nft.assetType == AssetType.Image)
+                                        GestureDetector(
+                                          onTap: () {
+                                            showDialog(
```

Comment:
i dont think we are using this way to show the dialog kindly check the codebase.